coconut/stubs/coconut/convenience.pyi

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# INFO:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+"""
+Author: Evan Hubinger
+License: Apache 2.0
+Description: Mypy stub file for convenience.py
+"""
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# IMPORTS:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+from typing import Any, Callable, Dict, Iterable, Text, Union
+
+from coconut.command.command import Command
+
+
+#-----------------------------------------------------------------------------------------------------------------------
+# COMMAND:
+#-----------------------------------------------------------------------------------------------------------------------
+
+
+CLI = ... # type: Command
```

Comment:
This is a stub file, so no need to use `# type: ` comments. Just write `CLI: Command = ...` instead.